service/src/test/java/bio/terra/workspace/common/fixtures/ControlledAwsResourceFixtures.java

```diff
@@ -250,6 +232,18 @@ public static String getUniqueNotebookName() {
         .region(AWS_REGION);
   }
 
+  public static ControlledAwsSageMakerNotebookResource makeResource(
```

Comment:
nit: can these names be more specific since they were moved to a shared fixtures files (e.g. makeAwsSagemakerNotebookResource?) Same for the folder above

Reply:
Sure, done